qgispluginci/utils.py

```diff
@@ -13,6 +13,12 @@ def replace_in_file(file_path: str, pattern, new: str, encoding: str = "utf8"):
         f.write(content)
 
 
+def add_in_file(file_path: str, new_line: str):
+    """ Add a line in a file. """
+    with open(file_path, "a", encoding="utf8") as f:
+        f.write("\n{}\n".format(new_line))
```

Comment:
```suggestion
        f.write("{}\n".format(new_line))
```
I'd prefer it like that to avoid unexpected blank lines, but no biggie.